.github/workflows/php-tests-wp-4-7-php-5-6.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
       - name: Composer Install
-        run: docker run --rm -v "$PWD:/app" -v "${{ steps.composer-cache.outputs.dir }}:/tmp/cache" composer install --no-interaction --no-progress # Run inside docker so composer install doesn't error...
+        run: docker run --rm -v "$PWD:/app" -v "${{ steps.composer-cache.outputs.dir }}:/tmp/cache" composer:2.1.11 install --no-interaction --no-progress # Run inside docker so composer install doesn't error...
```

Comment:
Looks like the newest version of Composer (released ~5 hours ago) contains backward-incompatible changes that cause a fatal error:

![image](https://user-images.githubusercontent.com/873049/144123387-31914c19-a74f-4d27-9076-a45a2a499368.png)

So, I have to update this command to make sure we use the latest compatible version of the `compose` image.